src/main/java/com/elevenpaths/tacyt/BaseSdk.java

```diff
@@ -348,4 +349,22 @@ protected final String getCurrentUTC() {
         return sdf.format(new Date());
 
     }
+
+    private String sortQueryString(String queryString) {
+        if (queryString == null || queryString.isEmpty()) {
+            return "";
+        }
+
+        if (queryString.contains("?")) {
+            String path = queryString.substring(0, queryString.indexOf("?") + 1);
```

Comment:
Why you add one?

Reply:
to mantein "?" character as part of the path String